<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1082</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>40</y>
      <width>991</width>
      <height>531</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_3">
     <widget class="QPushButton" name="pushButton_g_m">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>130</y>
        <width>281</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>gestion magasin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_g_p">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>130</y>
        <width>321</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>gestion parking</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page">
     <widget class="QLineEdit" name="lineEdit_id_e">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>nom magasin</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nbv_e">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>170</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_a_m">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>410</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>afficher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_e_ajouter">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>320</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_del_m">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>370</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>230</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>surface</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nom_e">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>120</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_m">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>30</y>
        <width>561</width>
        <height>281</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ID magasin</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>170</y>
        <width>63</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>adresse</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_spon_e">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>230</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>271</width>
        <height>281</height>
       </rect>
      </property>
      <property name="title">
       <string>ajouter/modifier</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_m_m">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>320</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>modifier</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>340</y>
        <width>241</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string>recherche</string>
      </property>
      <widget class="QLineEdit" name="lineEdit_sea_id">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>40</y>
         <width>113</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_sea_nom">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>90</y>
         <width>113</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_sea_sur">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>140</y>
         <width>113</width>
         <height>28</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>NOM</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>63</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>SURFACE</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>340</y>
        <width>141</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string>tri</string>
      </property>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>NOM</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>140</y>
         <width>83</width>
         <height>29</height>
        </rect>
       </property>
       <property name="text">
        <string>SURFACE</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_m_pdf">
      <property name="geometry">
       <rect>
        <x>880</x>
        <y>370</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>generer pdf</string>
      </property>
     </widget>
     <zorder>groupBox</zorder>
     <zorder>lineEdit_id_e</zorder>
     <zorder>label_2</zorder>
     <zorder>lineEdit_nbv_e</zorder>
     <zorder>pushButton_a_m</zorder>
     <zorder>pushButton_e_ajouter</zorder>
     <zorder>pushButton_del_m</zorder>
     <zorder>label_4</zorder>
     <zorder>lineEdit_nom_e</zorder>
     <zorder>tableView_m</zorder>
     <zorder>label</zorder>
     <zorder>label_3</zorder>
     <zorder>lineEdit_spon_e</zorder>
     <zorder>pushButton_m_m</zorder>
     <zorder>groupBox_2</zorder>
     <zorder>groupBox_3</zorder>
     <zorder>pushButton_m_pdf</zorder>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QLineEdit" name="lineEdit_p_id">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>120</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_p_nom">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_p_nbrp">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>230</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_p_idM">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>290</y>
        <width>113</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_p">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>40</y>
        <width>651</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_p_a">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>350</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_p_del">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>400</y>
        <width>81</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_p_aff">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>450</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>afficher</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>880</x>
      <y>0</y>
      <width>83</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>return</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1082</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
